babel.config.cjs

```diff
@@ -0,0 +1,6 @@
+module.exports = function (api) {
+  api.cache(true);
+  return {
+    presets: ["babel-preset-expo"],
+  };
+};
```

Comment:
Just curious. Wasn't this an `async` function before? Wondering if the `.cjs` is really necessary if this is a synchronous function

Reply:
No, it was never an `async` function. This file hasn't changed since the initial commit. The error message was complaining about something akin to, if you are going to use a an `es module` instead of a `commonjs module` that it has to be asynchronous. Non of the fixes I read about converted this function to an `async` function.

This is what the babel config docs say:
`babel.config.cjs and .babelrc.cjs allow you to define your configuration as CommonJS, using module.exports. They have been supported since v7.7.0.`